spec/index.bs

```diff
@@ -2254,7 +2254,8 @@ handling should allow it to recover.
 Any {{GPUAdapter}} returned by {{GPU/requestAdapter()}} must provide the following guarantees:
 
 - At least one of the following must be true:
-    - {{GPUFeatureName/"texture-compression-bc"}} is supported.
+    - Both {{GPUFeatureName/"texture-compression-bc"}} and
+        {{GPUFeatureName/"texture-compression-bc-sliced-3d"}} are supported.
     - Both {{GPUFeatureName/"texture-compression-etc2"}} and
         {{GPUFeatureName/"texture-compression-astc"}} are supported.
```

Comment:
```suggestion
        {{GPUFeatureName/"texture-compression-astc"}} are supported.
- If either {{GPUFeatureName/"texture-compression-bc"}} or
    {{GPUFeatureName/"texture-compression-bc-sliced-3d"}}
    is supported, both must be supported.
```

Reply:
@kainino0x This suggestion got outdated, but, do we need to add this? The reverted form seems more flexible and symmetrical, which might be a benefit if an edge case comes up. Thank you.

Reply:
I believe we need at least this one yes - see other comment for the reason. The suggestion in the other comment is optional though.

Reply:
>  it would allow devices which have etc2+astc to have bc without bc-sliced-3d or vice versa. Added a rule for this below.

Reply:
Tried to apply!